server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateMultiTimeSeriesPlan.java

```diff
@@ -0,0 +1,284 @@
+  private List<PartialPath> paths;
+  private List<TSDataType> dataTypes;
+  private List<TSEncoding> encodings;
+  private List<CompressionType> compressors;
+  private List<String> alias;
+  private List<Map<String, String>> props = null;
+  private List<Map<String, String>> tags = null;
+  private List<Map<String, String>> attributes = null;
+
+  /*
+   ** record the result of creation of time series
+   */
+  private Map<Integer, Boolean> results = new HashMap<>();
+  private List<Integer> indexes;
+
+  public CreateMultiTimeSeriesPlan() {
+    super(false, Operator.OperatorType.CREATE_MULTI_TIMESERIES);
+  }
+
+  @Override
+  public List<PartialPath> getPaths() {
+    return paths;
+  }
+
+  public void setPaths(List<PartialPath> paths) {
+    this.paths = paths;
+  }
+
+  public List<TSDataType> getDataTypes() {
+    return dataTypes;
+  }
+
+  public void setDataTypes(List<TSDataType> dataTypes) {
+    this.dataTypes = dataTypes;
+  }
+
+  public List<TSEncoding> getEncodings() {
+    return encodings;
+  }
+
+  public void setEncodings(List<TSEncoding> encodings) {
+    this.encodings = encodings;
+  }
+
+  public List<CompressionType> getCompressors() {
+    return compressors;
+  }
+
+  public void setCompressors(List<CompressionType> compressors) {
+    this.compressors = compressors;
+  }
+
+  public List<String> getAlias() {
+    return alias;
+  }
+
+  public void setAlias(List<String> alias) {
+    this.alias = alias;
+  }
+
+  public List<Map<String, String>> getProps() {
+    return props;
+  }
+
+  public void setProps(List<Map<String, String>> props) {
+    this.props = props;
+  }
+
+  public List<Map<String, String>> getTags() {
+    return tags;
+  }
+
+  public void setTags(List<Map<String, String>> tags) {
+    this.tags = tags;
+  }
+
+  public List<Map<String, String>> getAttributes() {
+    return attributes;
+  }
+
+  public void setAttributes(List<Map<String, String>> attributes) {
+    this.attributes = attributes;
+  }
+
+  public List<Integer> getIndexes() {
+    return indexes;
+  }
+
+  public void setIndexes(List<Integer> indexes) {
+    this.indexes = indexes;
+  }
+
+  public Map<Integer, Boolean> getResults() {
+    return results;
+  }
+
+  public void setResults(Map<Integer, Boolean> results) {
+    this.results = results;
+  }
+
+  @Override
+  public void serialize(DataOutputStream stream) throws IOException {
+    int type = PhysicalPlanType.MULTI_CREATE_TIMESERIES.ordinal();
+    stream.write(type);
+    stream.writeInt(paths.size());
+
+    for (PartialPath path : paths) {
+      putString(stream, path.getFullPath());
+    }
+
+    for (TSDataType dataType : dataTypes) {
+      stream.write(dataType.ordinal());
+    }
+
+    for (TSEncoding encoding : encodings) {
+      stream.write(encoding.ordinal());
+    }
+
+    for (CompressionType compressor : compressors) {
+      stream.write(compressor.ordinal());
+    }
+
+    for (String name : alias) {
```

Comment:
alias maybe null, please change the logic here.

Reply:
ok